pymeasure/instruments/yokogawa/aq6370d.py

```diff
@@ -155,24 +155,36 @@ def set_level_position_to_max(self):
     active_trace = Instrument.control(
         ":TRACe:ACTive?",
         ":TRACe:ACTive %d",
-        "Control the active trace (str 'TRA', 'TRB', 'TRC', ...).",
+        "Control the active trace (str 'A', 'B', 'C', ...).",
     )
 
     def copy_trace(self, source, destination):
         """
         Copy the data of specified trace to the another trace.
 
-        :param source: Source trace (str 'TRA', 'TRB', 'TRC', ...).
-        :param destination: Destination trace (str 'TRA', 'TRB', 'TRC', ...).
+        :param source: Source trace (str 'A', 'B', 'C', ...).
+        :param destination: Destination trace (str 'A', 'B', 'C', ...).
         """
+        mapping = {"A": "TRA", "B": "TRB", "C": "TRC", "D": "TRD"}
+
+        if source in mapping:
+            source = mapping[source]
+        if destination in mapping:
+            destination = mapping[destination]
+
         self.write(f":TRACe:COPY {source},{destination}")
```

Comment:
You want to be compatible with "TRA" and "A" as source/destination?
Another option:
~~~python
       self.write(f":TRACe:COPY TR{source.replace('TR', '')},TR{destination.replace('TR', '')}")
~~~

or you could use `source[-1]` to just use the last character.